vips/vip-499/bscmainnet.ts

```diff
@@ -0,0 +1,405 @@
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: DEFAULT_PROXY_ADMIN_ARBITRUM,
+        signature: "upgrade(address,address)",
+        params: [BOUND_VALIDATOR_ARBITRUM, BOUND_VALIDATOR_IMPLEMENTATION_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshot(uint256,uint256)", NORMAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setGrowthRate(uint256,uint256)", NORMAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshotGap(uint256)", NORMAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshot(uint256,uint256)", CRITICAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setGrowthRate(uint256,uint256)", CRITICAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshotGap(uint256)", CRITICAL_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshot(uint256,uint256)", FASTTRACK_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setGrowthRate(uint256,uint256)", FASTTRACK_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ACM_ARBITRUM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setSnapshotGap(uint256)", FASTTRACK_TIMELOCK_ARBITRUM],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: weETH_ORACLE_ARBITRUM,
+        signature: "setSnapshot(uint256,uint256)",
+        params: [increaseExchangeRateByPercentage(weETH_Initial_Exchange_Rate, BigNumber.from("44")), 1747682525],
```

Comment:
Given we have a snapshot gap, why are we increasing the exchange rate here (and in the other setSnapshot commands)? Is it to have some extra margin?

Reply:
The snapshot gap is only considered when the snapshot is done (function `updateSnapshot` ([here](https://github.com/VenusProtocol/oracle/blob/feat/transient-storage/contracts/oracles/common/CorrelatedTokenOracle.sol#L195))). In the `setSnapshot` function, the provided parameter is stored in the `snapshotMaxExchangeRate` variable without any modification, so we are "artificially" applying the gap that would be added if that value would have been stored in the `updateSnapshot` function